tests/testthat/test-lazy_functions.R

```diff
@@ -146,8 +146,8 @@ test_that("pl$n_unique", {
   expr_act = pl$n_unique("bob")
   expect_true(expr_act$meta$eq(pl$col("bob")$n_unique()))
 
-  expect_grepl_error(pl$n_unique(pl$all()))
-  expect_grepl_error(pl$n_unique(1))
+  expect_error(pl$n_unique(pl$all()))
+  expect_error(pl$n_unique(1))
```

Comment:
`expect_grepl_error()` is different from `expect_error()` because it can catch polars errors that have a weird structure in R.

If we don't check the pattern of the message, then `expect_error()` can be used indeed, but can you rather keep `expect_grepl_error()` and add some pattern matching?